src/styles/lists.styl

```diff
@@ -14,6 +14,9 @@
     grid-gap rem(10) 0
     justify-content center
 
+    &.detailed
+        grid-template-columns repeat(auto-fill, 15.5rem)
```

Comment:
How did you found this value ? maybe a comment would be great here to deconstruct the creation of the value

Reply:
indeed it's a magic number, I thought I would leave it for last, I'm adding a comment in the code to remind me of doing it